concordium-sdk/src/main/java/com/concordium/sdk/responses/nodeinfov2/BakerConsensusInfo.java

```diff
@@ -0,0 +1,70 @@
+package com.concordium.sdk.responses.nodeinfov2;
+
+import com.concordium.grpc.v2.NodeInfo;
+import com.concordium.sdk.responses.AccountIndex;
+import lombok.*;
+
+@Getter
+@Builder
+@EqualsAndHashCode
+@ToString
+public class BakerConsensusInfo {
```

Comment:
Please add one or two sentences of what the `BakerConsensusInfo` is. 
When is it relevant, when is it not relevant? What is it capturing? Etc.